lib/raven/processor.rb

```diff
@@ -2,11 +2,8 @@
 
 module Raven
   class Processor
-    attr_accessor :sanitize_fields
```

Comment:
perhaps I could generalize processor config, such that you can provide configuration keys to your own custom processor chain. I doubt anyone *really* writes their own processors though.